cypress/tests/ui/toolbar/global-toolbar.cy.js

```diff
@@ -33,7 +33,7 @@ describe('Global Toolbar', () => {
 
   it('verifies that users can change the theme from light to dark theme, or dark to light theme. #TC-3', () => {
     // Alias
-    cy.get('[data-test="Toggle Theme"]').as('toggleTheme');
+    cy.get('[data-test*="global-toolbar-theme-btn-"]').as('toggleTheme');
```

Comment:
Is it expected to keep `global-toolbar-theme-btn-` with `-` at the end here and at other places as well ?